.github/workflows/iroha2-release.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - name: Build wasm libs
-        run: ./scripts/build_wasm.sh libs
+        run: ./scripts/build_wasm.sh --profile=deploy libs
```

Comment:
Forgot to replace with `--release`?

Reply:
Ah, I see the `deploy` profile

Reply:
`--profile=release` is the default. `--profile=deploy` will be used for deploy.

Reply:
this is correct